apps/backend/src/resolvers/mutations/AssignProposalsToInstrumentsMutation.ts

```diff
@@ -67,4 +67,52 @@ export class AssignProposalsToInstrumentsMutation {
       args
     );
   }
+
+  @Mutation(() => Boolean)
+  async assignXpressProposalsToInstruments(
```

Comment:
I had another look and I think this might be the only resolver with validation, everywhere else does it in the mutation. From a technical standpoint it seems it can be done either way and this looks good, but I think we should ideally follow our existing convention